setup/bindir/cloud-setup-databases.in

```diff
@@ -248,21 +252,21 @@ for full help
             p = os.path.join(self.dbFilesPath, 'server-setup.sql')
             text = open(p).read()
             self.info("Applying %s"%p)
-            self.runMysql(text, p, True)
+            self.runMysql(text, p, self.rootuser != None)
             self.info(None, True)
 
         for f in ["templates"]:
             p = os.path.join(self.dbFilesPath,"%s.sql"%f)
             text = open(p).read()
             self.info("Applying %s"%p)
-            self.runMysql(text, p, True)
+            self.runMysql(text, p, self.rootuser != None)
```

Comment:
@Spaceman1984 what's the behavior when both 'schema-only' and 'deploy-as' options are passed to the script, and have valid values?

Reply:
The database creation, and user creation is skipped and the root user and password is used.

Reply:
I think, it is better to allow only one option

_cloudstack-setup-databases cloud:<dbpassword>@<host>:<post> [ --schema-only | --deploy-as=root:<password> ]_

Reply:
ok, I'll make it fail if you pass both.

Reply:
Done